python_modules/dagster/dagster/core/definitions/__init__.py

```diff
@@ -140,6 +140,7 @@
 from .solid_container import create_execution_structure
 from .solid_definition import CompositeSolidDefinition, NodeDefinition, SolidDefinition
 from .time_window_partitions import (
+    TimeWindow,
```

Comment:
Hey @nvinhphuc - it looks like the import ordering is off. You can fix this by running "make isort", or by moving this to its alphabetical position.

Reply:
Hi @sryza , I sorted the import, please check again if anything is wrong.

Reply:
Looks like that fixed it.